### ISSUE TITLE
Wrong render in table with new sdk 1.24.2 

### ISSUE THREAD

**Issue by Hecatoncheir** [bwu-dart/bwu_datagrid#155]
Hi. With new sdk **1.24.2** in chrome browser version **58.0.3029.110** I get wrong render in table:
![screenshot from 2017-07-10 16-03-11](https://user-images.githubusercontent.com/910917/28019410-85a1ed18-6589-11e7-87de-d66ebb21ea7a.png)

In pubspec I use bwu_datagrid version:  **'^0.1.0-dev.0'** 
Can this somehow be corrected?

**Comment by zoechi** [comment]
Thanks for the report.
I hadn't expected it to work or to be used anymore because Dart Polymer is not supported anymore.
Up to what Dart version did it work for you?
Are you using it with Polymer or polymerize?
Do you get an exception output in the browser console?
Have you tried in Chrome in addition to Dartium?

**Comment by Rasarts** [comment]
It work up to 1.23.0 version of dart.
I use Polymer in my pubspec:
```
dependencies:
 bwu_datagrid: '^0.1.0-dev.0'
 polymer: '1.0.0-rc.16'
```
In dartium it's work pretty well. In chrome I do not get errors in console.

**Comment by zoechi** [comment]
So the render issue is only in Chrome, but not in Dartium?

**Comment by Rasarts** [comment]
> So the render issue is only in Chrome, but not in Dartium?

Yes that's right

**Comment by zoechi** [comment]
It's just a guess that support for `/deep/` was finally removed from Chrome. I don't know about a workaround.

**Comment by Rasarts** [comment]
But it's work with 1.23.0 sdk and Chrome

**Comment by zoechi** [comment]
I see. That's weird.

**Comment by zoechi** [comment]
I don't think I'll be able to look at the issue soon. Did you investigate what could have caused the changed behavior?

**Comment by Rasarts** [comment]
No, I just updated the SDK version. I have no ideas with what it could be caused.

**Comment by Rasarts** [comment]
With SDK version 1.24.2 all works fine.
